docs/query-metadata-style-guide.md

```diff
@@ -179,8 +179,13 @@ The select clause of each alert query defines the alert message that is displaye
 * The message should factually describe the problem that is being highlighted–it should not contain recommendations about how to fix the problem or value judgements.
 * Program element references should be in 'single quotes' to distinguish them from ordinary words. Quotes are not needed around substitutions (`$@`).
```

Comment:
This seems to be in conflict with your proposed "Don't quote substitutions" rule. This says they're not *needed*, while your implementation says they're not *allowed*.

Reply:
Not quite. 
Quotes should be used around program text, when that program text is not a substitution.  

E.g. (for a simple query telling users to use StringBuilder instead of string concat). 
```CodeQL
select concat, "Use 'new StringBuilder()' instead of concatenating a large number of strings."
```

Reply:
👍